LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 
```

Comment:
Borrowing from Go's LICENSE, this could be:

`Copyright (c) 2016 The node-influx authors`